src/index.js

```diff
@@ -735,25 +777,26 @@ async function processIcons(components, fileId, config) {
           // Convert to vector drawable XML
           const xmlContent = await convertSvgForPlatform(optimizedSvg, 'android');
           const drawablePath = path.join(config.icons.path, 'drawable');
-          await fs.ensureDir(drawablePath);
+          await cleanAssetDirectory(drawablePath);
```

Comment:
Suggestion: We will use existing figma asset folder, this will cause all of our asset being cleared, to prevent conflict, maybe ask user that he want to override or not?

Reply:
Or maybe add a config to explicitly decide if we clear the target folder or not

Reply:
Updated - didn't know Android put all assets of the same dpi in the same place. Updated to not clear the directory.